### ISSUE TITLE
terraform-ls stopping with `missing required content-length`

### ISSUE THREAD

**Issue by navsal** [hashicorp/terraform-ls#651]
### Server Version

```
0.22.0
```

### Terraform Version

```
Terraform v1.0.7
on darwin_amd64
```

### Client Version

```
VSCode 1.60.2
```

### Command

`terraform-ls serve`

### Log Output

```
2021/09/25 19:06:25 serve_command.go:112: Terraform executions will be logged to /tmp/tf-exec-1-1.log (interpolated at the time of execution)
2021/09/25 19:06:25 serve_command.go:114: [WARN] -tf-log-file is deprecated in favor of `terraformLogFilePath` LSP config option
2021/09/25 19:06:25 serve_command.go:162: Starting terraform-ls 0.22.0
2021/09/25 19:06:25 service.go:83: Preparing new session ...
2021/09/25 19:06:25 langserver.go:94: Starting server (pid 9365; concurrency: 4) ...
2021/09/25 19:06:45 server.go:522: Server signaled to stop with err=missing required content-length
2021/09/25 19:06:45 server.go:147: Error reading from client: missing required content-length
2021/09/25 19:06:45 service.go:473: session stopped unexpectedly (err: missing required content-length)
2021/09/25 19:06:45 langserver.go:107: Stopping server (pid 9365) ...
2021/09/25 19:06:45 langserver.go:111: Server (pid 9365) stopped.
```

### Expected Behavior

I'm expecting `terraform-ls` not to crash.

### Actual Behavior

terraform-ls crashes, preventing me from being able to have IntelliSense on VSCode.

### Steps to Reproduce

1. Run `terraform-ls serve`

**Comment by radeksimko** [comment]
Hi @navsal 
Thank you for the report.

This looks like a relatedly low-level error.

A few questions, so we can get to the bottom of it:

1. Can you share some more details about the version of the Terraform VS Code extension? Hovering over [the relevant entry](https://user-images.githubusercontent.com/287584/134875580-3eabeff3-58b3-4021-85f6-47873dc29a29.png) in  Extensions pane should reveal it.
1. Can you confirm that the language server was automatically installed via the extension? if not - can you tell us whether you installed it via Homebrew or from any other source?
1. Can you either provide us a list of all other extension IDs currently installed, or confirm that this is reproducible for you in a VS Code instance with other extensions disabled? I just want to rule out a possibility that another extension is at fault.
1. Can you share any relevant settings options you set in VS Code starting with `terraform.`, `terraform-ls.` or `[terraform]`?
1. Can you try temporarily "downgrading" to the previous version of LS using `"requiredVersion": "0.21.0"` entry under `terraform.languageServer` settings in VS Code and check if the server is still being stopped with the same error? Just to confirm or rule out the possibility that this was introduced in the latest version.

**Comment by navsal** [comment]
Hi @radeksimko 

I managed to fix the problem by disabling and then reenabling all the extensions.

The output shared was from running the Language Server itself `terraform-ls serve` from the terminal, this language server was installed via _Homebrew_.

I've since then uninstalled the Language Server and reconfigured my VSCode setting to use the one bundled with the Extension.

This was most likely caused by other Terraform extensions I had installed in the past.

Thank you for your help.
